### ISSUE TITLE
Minior's core form shiny sprite is highly inaccurate

### ISSUE THREAD

**Issue by OpenSauce04** [talwat/pokeget-rs#11]
When attempting to show Minior's shiny sprite, the same colour as the non-shiny version is shown with different shading instead of the expected black core:

<img src=https://github.com/talwat/pokeget-rs/assets/48618519/32fcef6b-0b7e-4228-b323-545dfa5ee7b4 height=400px/>
<img src=https://github.com/talwat/pokeget-rs/assets/48618519/85c8d4c1-81cf-4264-8328-c92684af5e5b height=400px/>

**Comment by OpenSauce04** [comment]
This appears to be a holdover from the upstream pokesprite repo. The [gen 7 sprite](https://github.com/msikma/pokesprite/blob/master/pokemon-gen8/shiny/minior-blue-gen7.png) in the same repository seems to be coloured correctly; Is this just how shiny Minior's sprite looks in gen 8? Seems like a strange change

**Comment by talwat** [comment]
> This appears to be a holdover from the upstream pokesprite repo.
> 
> 
> 
> For whatever reason, there is [a correct shiny sprite](https://github.com/msikma/pokesprite/blob/master/pokemon-gen8/shiny/minior-blue-gen7.png) in the same respository with a `-gen7` suffix despite not being in the gen 7 folder. Any idea why this is the case?

No idea, but if that's the case then it'll be almost impossible to fix on the pokeget side, since it's not worth forking the whole repo to fix a bug. For now, it'll probably have to be stuck like this, but when they fix it i'll be able to pull the changes into pokeget.

**Comment by OpenSauce04** [comment]
In the meantime I have discovered that you can still get the correct sprite by typing out the full correct file name, which works in my case because I'm running from a script.
<img src=https://github.com/talwat/pokeget-rs/assets/48618519/e0632735-36cc-4c5a-84da-d7661686e9cc width=300px/>

**Comment by talwat** [comment]
Yeah, I made pokeget pretty flexible when it comes to how you're actually picking the sprites, so for scripts it should work fine :)

I'll keep this open to remind myself to fix this issue.
